Introspect/ViewExtensions.swift

```diff
@@ -73,6 +73,12 @@ extension View {
         return introspect(selector: TargetViewSelector.ancestorOrSibling, customize: customize)
     }
     
+    /// Finds a `UITableViewCell` from a `SwiftUI.List`, or `SwiftUI.List` child. You can attach this directly to the element inside the list.
+    public func introspectTableViewCell(customize: @escaping (UITableViewCell) -> ()) -> some View {
```

Comment:
I don't think using `.introspectTableViewCell()` on a `List` itself makes sense, because it'll only be introspecting one child. Better to target the cell directly – what do you think?

We can't disallow this from our code I believe, but maybe remove the mention of `from a SwiftUI.List`?

Reply:
You're right, it can't be disallowed explicitly, but there are good arguments for doing this from a list.

Attaching directly to the element within the list may give the developer the impression that they can modify the cell based on the current item in their iteration. But it doesn't work like that. UITableView creates cells on demand and while each cell will still have the customisations called on it, they will not match up with the items. This was the first thing I tried. Each item in my list had a color so I tried to set the select color to match. The results were completely random.

I've gone back and run some visual tests on ways of doing this and it turns out that putting the modifier on a list yields no visual changes. Putting it on the ForEach uniformly changes every cell. Putting it on the contents of the List (or ForEach) modifies every cell but not inline with the current item in the iteration.

Frankly, since the only way to really use this is to uniformly modify each cell, I would suggest that we demonstrate it on a ForEach within a List.